mypy/semanal.py

```diff
@@ -4310,3 +4334,21 @@ def find_fixed_callable_return(expr: Expression) -> Optional[CallableType]:
             if isinstance(t.ret_type, CallableType):
                 return t.ret_type
     return None
+
+
+def unmark_any_as_explicit(t: Type) -> None:
+    """For all types within t, if that type is Any, set explicit to False"""
+    t.accept(UnmarkAnyAsExplicit())
+
+
+class UnmarkAnyAsExplicit(TypeQuery[None]):
```

Comment:
This is a bit of an abuse of the `TypeQuery` class.  None of its parent classes seem to be set up for doing this easily, but I'm not entirely sure how I feel about this overall.  @JukkaL do you think this is a reasonable thing to do?